src/Storage/RrdDistributedStorage.php

```diff
@@ -129,4 +130,15 @@ private function copyRrdFiles(Device $device, StorageNode $from, StorageNode $to
             throw new \RuntimeException($error);
         }
     }
+
+    public function cleanup(CleanupService $cleanupService){
+
+        $storageNodes = $this->entityManager->getRepository(StorageNode::class)->findAll();
```

Comment:
What about disabled / inactive StorageNodes?
Can the repository dependency be injected instead? `getRepository()` is a service locator (an anti-pattern) and should be avoided when possible.